include/visit_struct/visit_struct.hpp

```diff
@@ -141,8 +252,7 @@ struct visitable<STRUCT_NAME, void> {
     using self_type = STRUCT_NAME;                                                                 \
     VISIT_STRUCT_PP_MAP(VISIT_STRUCT_MEMBER_HELPER_TYPE, __VA_ARGS__)                              \
   }                                                                                                \
-                                                                                                   \
-  static constexpr bool value = true;                                                              \
+  static VISIT_STRUCT_CONSTEXPR const bool value = true;                                           \
```

Comment:
I see, if it's helpful I guess it's not hard to put all the `constexpr` behind some macro.

I have merged a version of your patch now, let me know what you think. It seems to work on appveyor but I probably should actually install in a VM later.